### ISSUE TITLE
External browser is not obeyed in version 0.19.4

### ISSUE THREAD

**Issue by ghost** [QuiteRSS/quiterss#1317]
I've been using an external script file as my browser for years, but after upgrading to version 0.19.4 this stopped working, and only the system default browser is launched.

Version: 0.19.4
OS: Debian Testing w/ xfce4 + i3wm

**Comment by Funcy-dcm** [comment]
Perhaps because Qt changed paths. The application did not find the old settings. 
See: `Menu - Tools - Options - Browser - Following external browser: `

**Comment by ghost** [comment]
I don't think so, I tried changing the value, I also tried a total re-install including removing all the settings, and starting over. Still doesn't work. I removed it now and installed 0.19.3 instead.

**Comment by Funcy-dcm** [comment]
Send quiterss.ini

**Comment by ghost** [comment]
[quiterss.ini.zip](https://github.com/QuiteRSS/quiterss/files/4750074/quiterss.ini.zip)

**Comment by Unknow0059** [comment]
There seems to be no QuiteRSS documentation that I could find, and the forum has no search function.
How do I go about, say, playing a youtube video from QuiteRSS? Be it internally or externally (I don't know if it supports internal video playback, but it appears not).
I imagine I need a media player (which I do have) or a script like the one you @elundmark described, but I've no idea how to go about implementing either of these things.

**Comment by ghost** [comment]
@Unknow0059 - It's a bit of work yes, but I can give you some advice: Create a script that fires off other scripts, depending on what you use it for. That way you can use that external browser script for all kinds of links. Also, make sure you handle overflow; quiterss fires off all links you select in the news list at the same time.
Use mpv as the media player and also install youtube-dl from the python repos (pip). That way you can play Youtube videos. That is if you're using Linux. I'm sure you can install both on Windows/Mac too, but I have no idea how to do that.

**Comment by ghost** [comment]
I tried reinstalling 0.19.4 again and now it worked, no idea what I did differently, unless perhaps it had to do with caching? I guess I'll close this issue for now. Thanks!

**Comment by ghost** [comment]
I apologise but I misunderstood the issue the first time, I guess I gave up too soon. I've now tried this again and my issue is _not_ what I first reported:

The issue is, in version 0.19.3 I had my external browser set as a script, that handles all links opened by QuiteRss. That __also included the keyboard shortcuts for Pocket, Facebook__ etc. Now in version 0.19.4, these keyboard shortcuts will not go through the script, but opens the default browser instead, which is not the behaviour (I) am used to. Before, __All links__ opened through whatever you set in the Options.

**Comment by Funcy-dcm** [comment]
Send screenshots for 0.19.3 and 0.19.4: Menu - Help - About - Version

**Comment by ghost** [comment]
There is nothing that will show the problem in a screenshot; my settings say "use my script as the browser" but my script is only used when I click links, not the keyboard shortcuts for the social links. Are you sure you havn't changed anything regarding how the social links work, since 0.19.3 ?
